packages/cardano-services/README.md

```diff
@@ -144,6 +144,15 @@ RABBITMQ_SRV_SERVICE_NAME=some-domain-for-rabbitmq \
 The _Docker images_ produced by the SDK and the _docker compose infrastructures_ (_mainnet_, _preprod_ and _local-network_) it includes are ready to be used in
 production environment.
 
+To use _mainnet_, `cardano-node` my require a huge amount of RAM; this can lead to be killed due to OOM. The solution may be platform dependant:
```

Comment:
```suggestion
Running `cardano-node` on _mainnet_ requires a minimum of 24GB RAM.
```

Keep it simple 😃 
https://github.com/input-output-hk/cardano-node/releases/tag/1.35.7

Reply:
31a7c161 should solve this